node-module-system/calculate.js

```diff
@@ -0,0 +1,20 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+if (process.argv[3] === 'plus') {
+  console.log('result:', add(+process.argv[2], +process.argv[4]));
+}
+
+if (process.argv[3] === 'minus') {
+  console.log('result:', subtract(process.argv[2], process.argv[4]));
```

Comment:
To be more explicit in your code, convert these to numbers here as well, like you did for `add`. It happens to work because JavaScript is very forgiving, but it's dangerous. You should be more precise with what data types your code is using.